tmt/steps/provision/testcloud.py

```diff
@@ -341,3 +342,9 @@ def destroy(self):
         if self.instance:
             self.info('instance', 'stopping', 'green')
             self.instance.stop()
+        if self.option('login'):
+            self.info('instance', 'login', 'green')
+            p = subprocess.Popen(
+                ['ssh'] + self.ssh_args + [self.ssh_user_host])
+            p.communicate()
+        self.instance.stop()
```

Comment:
Reinventing this in every step seems redundant to me.

Isn't this just a variant of `execute('sh')`?

Reply:
@pvalena we cannot use any function that indents the output

Reply:
Optionally, if it would make sense, we could also add support for disabling indent to `run()`.

Reply:
> 
> 
> Optionally, if it would make sense, we could also add support for disabling indent to `run()`.

It would be logical to have it "on" for `--interactive` as well. Please see my comment.